src/Adapter/DbTable/CredentialTreatmentAdapter.php

```diff
@@ -107,7 +104,17 @@ protected function authenticateCreateSelect()
      */
     protected function authenticateValidateResult($resultIdentity)
     {
-        if ($resultIdentity['laminas_auth_credential_match'] != '1') {
+        /**
+         * Starting with PHP 8.1.0 integers and floats in result sets will be returned using native PHP types instead
+         * of strings when using emulated prepared statements. To keep the behavior consistent with older versions of
+         * PHP, strict comparison of both types is used.
+         *
+         * @linkhttps://www.php.net/manual/en/migration81.incompatible.php#migration81.incompatible.pdo.mysql
```

Comment:
A space is missing and the language code in the URL can be omitted; the website automatically redirects.

```suggestion
         * @link https://www.php.net/manual/migration81.incompatible.php#migration81.incompatible.pdo.mysql
```